react/features/invite/components/DialInNumbersForm.js

```diff
@@ -1,7 +1,7 @@
 import Button from '@atlaskit/button';
```

Comment:
Testing locally, this component looks off in the invite modal with the copy button taking up half the width of the modal and the other half being the dropdown.